monkestation/code/modules/research/designs/medical_designs.dm

```diff
@@ -67,3 +66,16 @@
 	surgery = /datum/surgery/robot_healing/experimental
 	id = "surgery_heal_robot_upgrade_femto"
 	research_icon_state = "surgery_chest"
+
+/datum/design/cyberimp_sprinter
+	name = "Vacuole ligament system"
+	desc = "These cybernetic eyes will display a medical HUD over everything you see. Wiggle eyes to control."
```

Comment:
the design uses the med hud implant description instead of the vacuole one it seems
```suggestion
	desc = "Mechicanical servos in ones leg that increases their natural stride. Popular amongst parkour enthusiasts. You need to implant this in both of your legs to make it work."
```

Reply:
Aaaa Got wires crossed when looking into a science hud implant. Thanks